tokio-rustls/Cargo.toml

```diff
@@ -22,6 +22,7 @@ dangerous_configuration = ["rustls/dangerous_configuration"]
 early-data = []
 logging = ["rustls/logging"]
 tls12 = ["rustls/tls12"]
+secret_extraction = ["rustls/secret_extraction"]
```

Comment:
Nit, would you mind moving this above `tls12` so as to keep them in alphabetic order?

Reply:
updated